src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+promise1
+  .then(() => {
+    const resolveMessage = document.createElement('div');
+
+    resolveMessage.className = 'message';
+    resolveMessage.textContent = 'Promise was resolved!';
+
+    document.querySelector('body').append(resolveMessage);
+  });
+
+promise2
+  .catch(() => {
+    const rejectMessage = document.createElement('div');
+
+    rejectMessage.className = 'message error-message';
+    rejectMessage.textContent = 'Promise was rejected!';
+
+    document.querySelector('body').append(rejectMessage);
```

Comment:
You can create helper function that will add message - it will keep code cleaner
```suggestion
});

const showMessage = (textContent , className ) => {
  const message = document.createElement('div');

  message.className = className ;
  message.textContent = textContent ;

  document.querySelector('body').append(message);
};

promise1
  .then(() => {
    showMessage('Promise was resolved!', 'message')
  });

promise2
  .catch(() => {
    showMessage('Promise was rejected!', 'message error-message')
```